ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -1360,10 +1451,27 @@ keys:
               keys:
                 source_protocol:
                   type: str
+                  valid_values:
+                    - "attached-host"
+                    - "bgp"
+                    - "connected"
+                    - "dhcp"
+                    - "dynamic"
+                    - "isis"
+                    - "ospfv3"
+                    - "static"
+                    - "user"
```

Comment:
This _could_ be considered a breaking change, in case people use some "hacks" by adding extra stuff in this string. That was often done to handle missing keys from our side. So I suggest we comment these new `valid_values` out and add a `# TODO: AVD5.0 Add valid values and ensure we cover all options`.

Reply:
same for other valid values added on existing models. Adding valid values on new models is fine.

Reply:
```suggestion
                  # TODO AVD5.0: Add valid values and ensure we cover all options.
                  #valid_values:
                  #  - "attached-host"
                  #  - "bgp"
                  #  - "connected"
                  #  - "dhcp"
                  #  - "dynamic"
                  #  - "isis"
                  #  - "ospfv3"
                  #  - "static"
                  #  - "user"
```